DuckDuckGo/TabViewController.swift

```diff
@@ -925,11 +967,43 @@ extension TabViewController: WKNavigationDelegate {
         }
     }
     
+    /*
+     Some files might be previewed by webkit but in order to share them
+     we need to download them first.
+     This method stores the temporary download or clears it if necessary
+     */
+    private func setupOrClearTemporaryDownload(for navigationResponse: WKNavigationResponse) {
```

Comment:
I wonder if we can be more clear in the function name about the purpose of it. Maybe something along the lines of `makeTemporaryDownloadForPreviewedFile(with navigationResponse: ...)`?

Reply:
`makeTemporaryDownloadForPreviewedFile` doesn't make it clear that it's going to `nil` out the var in case the temporary reference shouldn't exist.  `make` gives the impression that there's also something coming out of it.

Reply:
Fair point, I agree.